xbmc/video/VideoDatabase.cpp

```diff
@@ -6260,8 +6299,8 @@ bool CVideoDatabase::GetYearsNav(const std::string& strBaseDir, CFileItemList& i
       std::string group;
       if (idContent == VIDEODB_CONTENT_MOVIES)
       {
-        strSQL = "select movie_view.premiered, count(1), count(files.playCount) from movie_view ";
-        extFilter.AppendJoin("join files on files.idFile = movie_view.idFile");
+        strSQL = "SELECT movie_view.premiered, COUNT(1), COUNT(history.idFile) FROM movie_view ";
+        extFilter.AppendJoin("JOIN history ON history.idFile = movie_view.idFile");
```

Comment:
Check if we can use subqueries here instead, as inner join might hide some values